content/applications/inventory_and_mrp/barcode/setup/hardware.rst

```diff
@@ -106,3 +106,60 @@ Once that option is enabled, go back to the :guilabel:`Profile` options page, an
 
 Once all those steps have been taken, perform a test scan to ensure the Zebra scanner is working
 properly, as intended.
+
+Honeywell mobile computer scanner
+=================================
+
+When using Honeywell scanners, follow the instructions below to ensure barcodes can be scanned in
+Odoo.
+
+Begin on the Honeywell scanner's home screen, and select :guilabel:`Settings`, represented by a
+:guilabel:`⚙️ (gear)` icon. Then, click :guilabel:`Honeywell Settings`, followed by
+:guilabel:`Scanning`.
+
+From here, click :guilabel:`Internal Scanner`, followed by :guilabel:`Default Profile`. From the
+resulting list of options, select :guilabel:`Data Processing Settings`.
+
+The :guilabel:`Data Processing Settings` specify how barcode data is processed by the computer.
+Locate the :guilabel:`Wedge Method` setting. By default, this is set to :guilabel:`Standard`.
+
+.. image:: hardware/hardware-honeywell-settings.png
+   :align: center
+   :alt: Data processing settings options for Honeywell scanner.
+
+Change the :guilabel:`Wedge Method` setting to :guilabel:`Keyboard`.
+
+After completing the steps, conduct a test scan to verify the Honeywell scanner is working as
+intended.
+
+Cipherlab mobile computer scanner
+=================================
+
+When using Cipherlab scanners, follow the instructions below to ensure barcodes can be scanned in
+Odoo.
+
+Begin on the Cipherlab scanner's home screen, and navigate to the :menuselection:`App Drawer (All
+Applications)`. Then, click the :guilabel:`ReaderConfig` app, represented by an orange
+:guilabel:`⚙️ (gear)` icon over a blue barcode icon.
```

Comment:
Small nitpick- add icon descriptor formatting around this too. I know this is an odd case since you are describing an app icon that is made up of multiple icons and colors.
```suggestion
:guilabel:`⚙️ (gear)` icon over a blue :guilabel:`(barcode)` icon.
```

Reply:
Suggestion implemented ✅